OrbitService/CaptureServiceImpl.cpp

```diff
@@ -137,7 +137,11 @@ grpc::Status CaptureServiceImpl::Capture(
   CaptureRequest request;
   reader_writer->Read(&request);
   LOG("Read CaptureRequest from Capture's gRPC stream: starting capture");
+
   tracing_handler.Start(std::move(*request.mutable_capture_options()));
```

Comment:
You stop using different threads. Do you also want to do this here for the start?

Reply:
The reason why I'm stopping using different threads is that the "stop" methods are blocking, waiting for remaining events (I'll add a comment there). These "start" methods are not blocking hence why they are in the same thread.

Reply:
Okay, make sense. Then there should be probably a comment for that on the stop part.

Reply:
Yes, that's there.